modules/nf-core/merfin/hist/tests/modules.config

```diff
@@ -0,0 +1,17 @@
+process {
+	withName: MERYL_COUNT {
+		ext.args = { "k=6" }
+	}
+
+	withName: MERYL_HISTOGRAM {
+		ext.args = { "k=6" }
+	}
+
+	withName: GENOMESCOPE2 {
+		ext.args = { "--ploidy 1 --kmer_length 6 --fitted_hist" }
+	}
+}
+
+params {
+	modules_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/'
+}
```

Comment:
I notice that this shouldn't be here. This is already defined in tests/config/nf-test.config.

Reply:
It's true. Thanks for pointing this out. I've removed the line.